std/uni.d

```diff
@@ -7102,7 +7102,7 @@ public: // Public API continues
     Returns:
         length of grapheme cluster
 +/
-size_t graphemeStride(C)(const scope C[] input, size_t index)
+size_t graphemeStride(C)(const scope C[] input, size_t index) @safe pure
```

Comment:
After going all the way down the rabbit hole, I found out that due to some compiler bug, DMD marked this as `@system` when it was instantiated from `std.formatRange` from `std.conv.textImpl`. No idea why. Manually adding the attributes fixes the whole issue.